### ISSUE TITLE
Cannot run on M1 Macs

### ISSUE THREAD

**Issue by newmana** [zwilias/elm-coverage#41]
When running from the command line:
`yarn run coverage`

The following error is displayed:
```
Exit code: 1
Command: binwrap-install
Arguments:
Directory: ./node_modules/elm-coverage
Output:
./node_modules/binwrap/install.js:18
    throw new Error("No binaries are available for your platform: " + buildId);
    ^

Error: No binaries are available for your platform: darwin-arm64
```

**Comment by zwilias** [comment]
I don't think elm-instrument can be compiled for M1 yet; is it an option to rely on the x86 binary and rosetta? If so, I can make the changes and publish a patch!

**Comment by newmana** [comment]
I was testing this out this afternoon and this works with Rosetta 2 https://github.com/newmana/elm-coverage/blob/main/install.js

**Comment by newmana** [comment]
You don’t need to do anything else, btw.

**Comment by zwilias** [comment]
Nice, let's go with that! If you feel like making a PR, please do 👍

**Comment by newmana** [comment]
https://github.com/zwilias/elm-coverage/pull/42

**Comment by zwilias** [comment]
Resolved by #42

**Comment by zwilias** [comment]
And published in 0.4.1 🎉
